### ISSUE TITLE
Using Google Colab GPU to train model

### ISSUE THREAD

**Issue by sgbatman** [matterport/Mask_RCNN#2125]
Hi everyone,

I am using Google colab to train a custom model. However, Google colab keep prompting me that I am not using the GPU in the instance. Can anyone let me know where do I have to configure the codes to make the model training run on a GPU? Any help is appreciated thank you.

**Comment by susnato** [comment]
Can you please check if you are using a GPU as a hardware accelerator or not in notebook settings?

**Comment by sgbatman** [comment]
I already change the runtime in the notebook settings to GPU. But it is telling me to switch back to GPU cuz I'm not utilizing the GPU.

**Comment by susnato** [comment]
Ok Can you please run this code in colab and tell if tensorflow is on gpu or not ?
tf.test.is_gpu_available(
    cuda_only=False, min_cuda_compute_capability=None
)

**Comment by aghasaadmohammad** [comment]
!pip install keras==2.0.8
!pip install tensorflow-gpu==1.15.0
!pip install pycocotools
!pip install scikit-image
!pip install imgaug
Run these and try

**Comment by agbozo1** [comment]
To train your mask-rcnn model in colab, you can first setup the runtime as GPU and install these 3 libraries.
```
!pip install tensorflow-gpu==1.15.2
!pip install keras==2.1.5
!pip install h5py==2.10.0
```
